templates/demo/app/dapp.js

```diff
@@ -27,7 +28,7 @@ class App extends React.Component {
   }
 
   componentDidMount() {
-    EmbarkJS.onReady((err) => {
+    EmbarkJS.Blockchain.connect(config, (err) => {
```

Comment:
Apparently we have a breaking change, would it be the good time to also support promise if there is no callback?

Reply:
We do. I added support for promise AND callback